test/regression/dollar-sign-body-test.coffee

```diff
@@ -0,0 +1,33 @@
+{assert} = require 'chai'
+Gavel2Html = require('../../src/index')
+
+describe 'Dollar sign in the body', ->
+  gavelResult = {
+    fields: {
+      body: {
+        kind: 'json',
+        values: {
+          actual: {
+            symbol: "$"
+          },
+          expected: {
+          }
+        },
+        errors: []
+      }
+    }
+  }
+
+  payload = {
+    fieldName: 'body',
+    fieldResult: gavelResult.fields.body,
+  }
+  diff = new Gavel2Html payload
+  html = diff.getHtml({
+    missingStartTag: '<missing>',
+    addedStartTag: '<added>',
+    changedStartTag: '<changed>',
+  })
+
+  it 'should tolerate a dollar sign in the body', ->
+    assert.equal(html, '<li>{</li><added>&quot;symbol&quot;:&quot;$quot;</li><li>}</li>')
```

Comment:
Perhaps like this?

```suggestion
    assert.equal(html, '<li>{</li><added>&quot;symbol&quot;:&quot;$&quot;</li><li>}</li>')
```

Reply:
I'll update the test. It should actually be `$$&quote;`.